client/scss/components/_blockquote.scss

```diff
@@ -0,0 +1,54 @@
+.blockquote {
+  @include font('HNM2');
+  text-align: center;
+
+  &--wb {
+    @include font('WB5');
+  }
+
+  &--lr {
+    @include font('LR2');
+    text-align: left;
+  }
+
+  footer {
+    @include font('HNM5');
+
+    &:before {
+      content: '\2015';
+      margin-right: 0.5em;
+    }
+  }
+
+  p {
+    margin-top: 0;
+  }
+
+  cite {
+    @include font('HNL5');
+    font-style: normal;
+  }
+
+  &:before,
+  &:after {
+    // stylelint-disable sh-waqar/declaration-use-variable
+    // Decorative quotes – not defined as part of the typography styles
+    font-family: $helvetica-neue-medium-base;
+    font-size: 6rem;
+    // stylelint-enable sh-waqar/declaration-use-variable
+    line-height: 1;
+    display: block;
+    color: color('action-green-1');
+    height: 3rem;
+  }
+
+  &:before {
+    content: '\201c';
+    margin-bottom: 1rem;
+  }
+
+  &:after {
+    content: '\201d';
```

Comment:
It would be interesting to see what's best for screenreaders.

If it's in a quote, are we going to land up with a, "start blockquote, start quote, blahblahblah, end quote, end blockquote".